### DISCUSSION TITLE
Thino pro 激活设备数到上限了怎么办？能不能移除之前激活的设备？

### DISCUSSION THREAD

**Discussion by lj723911** [Quorafind/Obsidian-Thino#724] [Q&A 问答]
如题。

**Comment by juestchaos** [comment]
After the device activation limit is reached, if it is not due to innocence or the sale of Thino devices, you can email the community mailbox to report the issue, providing specific reasons and the devices to be removed. Note that if you have not reached the maximum number of devices, the community suggests waiting until you have used the full quota before taking further action.
